tests/lib/rules/require-expect.js

```diff
@@ -294,6 +305,35 @@ ruleTester.run("require-expect", rule, {
             code: "test('name', function(assert) { other.expect(1); assert.ok(true); });",
             options: [],
             errors: [{ message: alwaysErrorMessage("assert.expect") }]
+        },
+
+        // "always" configuration - errors are reported on `test()` line
```

Comment:
My intent here was to test that the error was reporting at the correct location. I believe the next test with `column 1` shows that it is using the correct location.